modal/image.py

```diff
@@ -994,8 +995,8 @@ def poetry_install_from_file(
         If not provided as argument the path to the lockfile is inferred. However, the
         file has to exist, unless `ignore_lockfile` is set to `True`.
 
-        Note that the root project of the poetry project is not installed,
-        only the dependencies. For including local packages see `modal.Mount.from_local_python_packages`
+        Note that the root project of the poetry project is not installed, only the dependencies.
+        For including local packages see `add_local_python_dir`
```

Comment:
I wonder if this is confusing since usually the root project would be what gets included through the modal source mount right? Or maybe not if we start disabling automount.

Reply:
Depends. If this is a *separate* python project from the current one, it wouldn't be included by automounts unless it's imported in global scope locally. If it's the project that the modal app's module is included in, it would be automounted if the modal app is is deployed/run via module path syntax, but not necessarily if it's included via `file/path.py`.

All of these inconsistencies would get somewhat fixed by removing automount except for the function *file* itself.

Reply:
At the expense of having to be a bit more explicit whenever your modal function references other local files 😬